Assets/Scripts/UI/PauseMenu/PauseMenuUI.cs

```diff
@@ -1,3 +1,19 @@
+using UnityEngine;
+using UnityEngine.UI;
+
 public class PauseMenuUI : UICanvasBase
 {
+    [SerializeField] private Button MainMenuButton;
+
+    private void Awake()
+    {
+        MainMenuButton.onClick.RemoveAllListeners();
+        MainMenuButton.onClick.AddListener(ShowMainMenuCanvas);
+    }
+
+    private void ShowMainMenuCanvas()
+    {
+        UIManager.ShowCanvas(canvasType: UICanvasType.MainMenu, viaShortcut: false);

```

Comment:
Looks good. 
Don't worry about this btw:
![image](https://user-images.githubusercontent.com/61809127/150439773-9b1f6b5b-00b9-47f4-8778-518326bcd025.png)
It's a slight issue with the canvas/player prefs that haven't got round to addressing. I don't think it actually makes any difference in practice in this case anyway.